io.openrct2.OpenRCT2.yaml

```diff
@@ -0,0 +1,37 @@
+app-id: io.openrct2.OpenRCT2
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: openrct2
+rename-appdata-file: openrct2.appdata.xml
+rename-desktop-file: openrct2.desktop
+rename-icon: openrct2
+copy-icon: true
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  # Pulseaudio access
+  - --socket=pulseaudio
+  # Filesystem access
+  # to access local images and other files
+  - --filesystem=home
+  # Network access
+  # to access remote images
+  - --share=network
```

Comment:
Does it actually need network access?

Reply:
I'm not sure - if I remove the flag, do you have time to try a build without it?

Reply:
Yeah, no problem.

Reply:
In OpenRCT2 we have (compile-time opt-out) multiplayer support and soon to be done update checking

Reply:
Aha, thanks. I'll leave network support in place, then.

How does it work with the game data files - does it copy them to a hidden data directory when you select them, or does it keep using them from the folder you pointed to? This might affect what filesystem access it needs.

Reply:
The RCT 2 assets are intact and used from their specified location. All the files we create are located inside "user data dir", by default `$XDG_CONFIG_HOME` or, if unset, `~/.config/OpenRCT2`. There's a command line switch which can be used to override this location. The files we store contain configuration, external objects, tracks, etc.

Reply:
Thanks. I think this should work with read-only access to the user's home directory (it will still have write access to its own XDG cache/config/home dirs).